tests/integration/test_modular_decomposition_isomorphisms.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+from color_refinement_helper import graph_to_modules, modules_to_graph_with_module_isomorphism
+from tools import create_graph_helper
+
+
+class Test(unittest.TestCase):
+    def test_moeilijkere_graaf(self):
+        g = create_graph_helper(
+            [(0, 1), (1, 2), (2, 3), (3, 4), (4, 5), (5, 6), (6, 7), (5, 7), (5, 8), (3, 8), (3, 9), (1, 9), (2, 9),
+             (1, 10), (6, 11), (7, 11), (11, 12), (11, 13), (11, 14), (12, 13), (12, 14), (13, 14)])
+
+        h = g.deepcopy()
+        md = graph_to_modules(g + h)
+        graph_md, modular_iso = modules_to_graph_with_module_isomorphism(md)
+
+        # Dit assert nu niks, maar deze graaf is wel een beginpunt
```

Comment:
Graag deze afmaken

Reply:
Bv
```
# Expect 3 groups of isomorphic modules
        # 1. 0+10, 0+10, 4+8, 4+8
        # 2. 2+9, 2+9, 6+7, 6+7
        # 3. 12+13+14, 12+13+14
        v0_10, v4_8, v2_9 = graph_md.find_vertex('0+10'), graph_md.find_vertex('4+8'), graph_md.find_vertex('2+9')
        v6_7, v12_13_14 = graph_md.find_vertex('6+7'), graph_md.find_vertex('12+13+14')
        self.assertEqual(3, len(modular_iso))
        expected_modules = [[v2_9, v2_9, v6_7, v6_7],[v12_13_14, v12_13_14],[v0_10, v0_10, v4_8, v4_8]]
        for i in range(3):
            self.assertTrue(compare(modular_iso[i], expected_modules[i], key=lambda v: v.label))
```

Reply:
En wat is de moduler_decompensation_factor van deze graaf? De functie zegt 9216, maar volgens mij is het veel lager.
Eg. modules [0+10, 4+8, 0+10, 4+8] kunnen op elkaar gemapt worden op 2 manieren en hebben elk 2 opties voor coloring -> 4
Hetzelfde voor [2+9, 6+7, 2+9, 6+7] -> 4
Modules [12+13+14, 12+13+14] kunnen maar op 1 manier gemapt worden en op 3! manier gekleurd worden.
Is de factor dan 4*4*3! = 96 ?

Reply:
Maar de samenvoeging van vertices in module levert ook al n-faculteit mogelijkheden op hè, kan het daar vandaan komen?

Reply:
Daar komt het inderdaad vandaan, zie Slack voor de verklaring. De disjoint union heeft MD factor te groot. De MD factor van de individuele grafen is 96 en dat is goed.